Codec_Zero_OnboardMIC_record_and_SPK_playback.state

```diff
@@ -1192,3 +1192,1199 @@ state.Zero {
 		}
 	}
 }
+
+state.IQaudIOCODEC {
```

Comment:
This looks like the content of the old IQAudio_Codec_AUXIN_record_and_HP_playback.state.